src/irc/BridgedClient.ts

```diff
@@ -740,7 +760,7 @@ export class BridgedClient extends EventEmitter {
             }
             // Add arbitrary letter prefix. This is important for guest user
             // IDs which are all numbers.
-            n = "M" + n;
+            n = "`" + n;
```

Comment:
Most IRCd's don't allow numbers and `-` at the start of nicknames.

> this is just for guest accounts so I wouldn't worry about it too much

I met a whole lot of people with numbers at the start of their nickname.

> I get the idea of the backtick being more IRC-natural, but 1) I'd expect it as a suffix rather than a prefix, so that usual autocompletion still works in clients

Not sure what you mean there.